samples/statbar/statbar.cpp

```diff
@@ -940,6 +939,12 @@ MyStatusBar::MyStatusBar(wxWindow *parent, long style)
 
     m_statbmp = new wxStaticBitmap(this, wxID_ANY, wxIcon(green_xpm));
 
+#ifndef __WXQT__
```

Comment:
I think we should get rid of this and use `AddFieldControl()` everywhere and just use an intermediate parent window that would be center the bitmap inside it.

Or is there some problem with doing this?

Using conditional compilation in the samples should really be avoided unless absolutely necessary IMO...